### PR TITLE
feat: APPS-2340 robots.txt, listing page for impact reports

### PR DESCRIPTION
**Pull request by sourcefilter**
Connected to [APPS-1903](https://jira.library.ucla.edu/browse/APPS-1903)
Connected to [APPS-2340](https://jira.library.ucla.edu/browse/APPS-2340)

https://deploy-preview-720--uclalibrary-test.netlify.app/impact/all
redirects to:
https://deploy-preview-720--uclalibrary-test.netlify.app/about/reports/

https://deploy-preview-720--uclalibrary-test.netlify.app/impact/
redirects to the current years impact report
https://deploy-preview-720--uclalibrary-test.netlify.app/impact/

### CODE REVIEW

File: pages/about/reports/index.vue [hunk 1]
```diff
@@ -3,16 +3,33 @@
...
+        <section-wrapper>
+            <section-staff-article-list :items="parsedReportsList" />
+        </section-wrapper>
+
+        <section-wrapper theme="divider">
+            <divider-way-finder
+                color="about"
+            />
+        </section-wrapper>
     </main>
```
Comment (axamei) [inline pages/about/reports/index.vue:33]: Let's add the global call to action block:
```
        <section-wrapper>
            <block-call-to-action
                class="block-call-to-action"
                :is-global="true"
            />
        </section-wrapper>
```

File: static/_redirects [hunk 2]
```diff
@@ -210,7 +210,8 @@
 /help/services-resources/research-data-services/	/help/services-resources/data-services
 /help/services-resources/teaching-with-special-collections	/help/services-resources/teaching-with-library-special-collections
 /hours	/visit/locations
-/impact	/impact/all
+/impact	/about/reports
+/impact/all	/about/reports
```
Comment (axamei) [inline static/_redirects:214]: I am getting a 404 when I click on the links. Do redirects work in test?

**Review by axamei — CHANGES_REQUESTED** [review]
Getting 404s on link, but that might be an env issue. Add the global call to action block and this should be good to go. Thanks!

**Review by axamei — DISMISSED** [review]

**Review by jendiamond — APPROVED** [review]
🥮

**Pull request merged**